lib/Terminal/Widgets/Input/Menu.rakumod

```diff
@@ -17,7 +17,7 @@ class Terminal::Widgets::Input::Menu
     submethod TWEAK() {
         self.Terminal::Widgets::Input::TWEAK;
         for $!items.kv -> $i, $item {
-            %!hotkey{$_} = $i for ($item<hotkeys> // []).words;
+            %!hotkey{$_} = $i for $item<hotkeys>[];
```

Comment:
What failure mode did you see here?  The version I wrote handled a number of cases I tried, but maybe I'm just thinking of something different from what you had in mind?